src/__type_tests__/chip-type-test.tsx

```diff
@@ -62,3 +64,9 @@ import Chip from '../chip';
 <Chip to="/" onClose={() => {}}>
     hello
 </Chip>;
+
+const ChipMaybeOnPress = ({text, onPress}: {text: string; onPress?: PressHandler}): JSX.Element => (
+    <Chip maybe onPress={onPress}>
```

Comment:
It feels weird to see something like `<Chip maybe/>`. Perhaps we could consider something like `maybeTouchable` or `maybeInteractive`.

Also, this would require to add `maybe` in all the problematic scenarios. I'm not sure this really solves the issue. I think leaving it as it is won't be a big issue.

Let's discuss this a bit better in the next weekly